Add some spiral examples

autotunafish opened the pull request:
Sketches for a logarithmic spiral, a logarithmic 3d effect spiral, Fibonacci spiral and the Ulam Prime spiral as well as some sample images

autotunafish committed:
Add files via upload

Images of example output

autotunafish committed:
Delete fibspiral.svg

autotunafish committed:
Delete ulamspiral.svg

autotunafish committed:
Delete three_d_logspiral.svg

autotunafish committed:
Delete logspiral.svg

autotunafish committed:
Add files via upload

Images of example output

autotunafish committed:
Add files via upload

added spiral examples: ulamspiral, logspiral. three_d_spirals and fibonacci

sunjay commented:
Thanks for making this! I haven't been actively maintaining this project for a while so I won't have time to review in depth. I took a cursory look and it looks fine overall. Have another look at the diff and let me know when you're satisfied with it. I'll merge it in when you're ready. 

I don't think the build will pass right now but as long as you have it working locally it should be good to go.

autotunafish commented:
Ok, cool. I forked the repo and compiled and rendered those images yesterday in the current version. I had originally written them in the older style described on the turtle website and following that method only barely worked as it would panic randomly and often crash if I moved the mouse. So seeing the build failures here made sense but pulling down the latest version and after I had refactored the example code with the newer methods to work properly it actually ran (seemingly) pretty stable or at least  by comparison.

sunjay commented:
Yeah, the version on crates.io is years old at this point and the dependencies there don't work at all. #183 details why we haven't released the current version in the repo that fixes some of those problems (and introduces others).
